docs/inventory.md

```diff
@@ -372,6 +372,9 @@ If you need VMs with static UUIDs to allow them to be reused then the UUID for a
               uuid: d36ebda0-25bf-55ef-9c69-66ad5ef0d39d
 ```
 
+#### Bridged network vs. NATted network
+
+By default, this repository deploys a bridged network to interconnect all virtual machines created between them, but it is possible to use a NATted network instead. You just need to set `use_natted_network` to `true` (default `false`) and `use_bridged_network` to `false`(default `true`).
```

Comment:
What happens if people set both to false by accident, would this completely break the install? I think at very least we need an inventory validation to protect against this. 

And perhaps a single value to switch between the two would be better something like `vm_network_type`. Once again with an inventory validation to check that is set to an allowed value.